packages/perseus/src/widgets/interactive-graphs/mafs-styles.css

```diff
@@ -214,53 +216,9 @@
 .MafsView pattern g {
     stroke: rgba(33, 36, 44, 0.32);
 }
-.axis-tick-labels {
```

Comment:
All of these styles are no longer needed as we're using attributes to ensure proper compatibility with older versions of Safari.